smrt/interface/coherent_flat.py

```diff
@@ -71,7 +76,7 @@ def specular_reflection_matrix(self, frequency, eps_1, eps_2, mu1, npol):
             npol: Number of polarization.
 
         Returns:
-            The reflection matrix.
+            The reflection matrix, :py:mod:`smrt.core.fresnel.fresnel_reflection_matrix`.
```

Comment:
please remove :py:mod:`smrt.core.fresnel.fresnel_reflection_matrix`, this is not exactly what this function is doing